src/System.CommandLine.Tests/CompletionTests.cs

```diff
@@ -311,6 +311,7 @@ public void When_a_subcommand_has_been_specified_then_its_sibling_commands_will_
             var result = rootCommand.Parse("cherry ", simpleConfig);
 
             result.GetCompletions()
+                  .Select(item => item.Label)
```

Comment:
Nice catch! I wonder how it worked so far with `FluentAssertions` (using `.ToString` implicitly?)